### PR TITLE
add a new button style

### PR DESCRIPTION
**Pull request by beshoo1**
I want to stop using bootstrap styling system , so i add buttons style

### CODE REVIEW

**Comment by craftpip** [comment]
Thanks for the code, I'm planning to add this in the next version v3. im about to finish work with v3 now

File: css/jquery-confirm.css [hunk 1]
```diff
@@ -517,3 +517,73 @@ body.jconfirm-noscroll {
   margin: 0;
   border-left: solid 1px #ddd;
 }
+
+
+.jcbutton {
```
Comment (craftpip) [inline css/jquery-confirm.css]: Thank you for this!

Comment (beshoo1) [inline css/jquery-confirm.css]: you are welcome,  did you add bootstrap free?

Comment (craftpip) [inline css/jquery-confirm.css]: Yes made bootstrap optional now, now I'm adding buttons now, with help of your CSS. Will push the changes in a few hours.

Comment (craftpip) [inline css/jquery-confirm.css]: Feature is added.

**Pull request closed without merging**